packages/orchestrator-ui-components/src/components/WFOSubscription/SubscriptionProductBlock.tsx

```diff
@@ -0,0 +1,151 @@
+                                                style={{
+                                                    width: 250,
+                                                    padding: 10,
+                                                    borderBottom: `solid ${isLastCell(
+                                                        index,
+                                                    )}px #ddd`,
+                                                }}
+                                            >
+                                                <b>
+                                                    {
+                                                        productBlockInstanceValue.field
+                                                    }
+                                                </b>
+                                            </td>
+                                            <td
+                                                style={{
+                                                    padding: 10,
+                                                    borderBottom: `solid ${isLastCell(
+                                                        index,
+                                                    )}px #ddd`,
+                                                }}
+                                            >
+                                                {
+                                                    productBlockInstanceValue.value
+                                                }
+                                            </td>
+                                        </tr>
+                                    ) : (
+                                        <tr key={index}>
+                                            <td
+                                                colSpan={3}
+                                                style={{
+                                                    padding: 10,
+                                                    borderBottom: `solid ${isLastCell(
+                                                        index,
+                                                    )}px #ddd`,
+                                                }}
+                                            >
+                                                <EuiAccordion
+                                                    id={
+                                                        productBlockInstanceValue.field
+                                                    }
+                                                    arrowDisplay="left"
+                                                    buttonContent={
+                                                        <div>
+                                                            <td
+                                                                valign={'top'}
+                                                                style={{
```

Comment:
It might be good to also clean up the inline css

Reply:
I propose doing this is a separate ticket. There are a bunch of other places where inline css is used so it would be better to do that all at once.